setup.cfg

```diff
@@ -1,2 +1,6 @@
 [flake8]
 max-line-length=99
+[tool:pytest]
+log_cli = True
+log_file = logs/pytest-logs.txt
```

Comment:
I left that in there from the branch that Bhavin made because I thought it might be helpful in local testing, but since everything is printed out in the terminal we should be good to remove it without losing anything. We can download log files directly from GHA if we want them locally and/or we could just do a pytest cli command to locally save a log file if we need to.